Dockerfile

```diff
@@ -44,14 +46,14 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 RUN ln -s -f /usr/bin/python3 /usr/bin/python
 
 # Install NCCL 2.2
-RUN apt-get install -qq -y --allow-downgrades --allow-change-held-packages libnccl2=2.3.7-1+cuda10.0 libnccl-dev=2.3.7-1+cuda10.0
+RUN apt-get --no-install-recommends install -qq -y --allow-downgrades --allow-change-held-packages libnccl2=2.3.7-1+cuda10.0 libnccl-dev=2.3.7-1+cuda10.0
```

Comment:
Have you verified that everything runs as intended without the recommended packages installed ?

Reply:
yes , it works , as recommended packages  are not required dependences , those are only addons

Reply:
@Rajpratik71 this is the last thing to fix / verify

Reply:
I know a little bit about Debian, thanks. It could still happen that runtime depends on some packages recommended.